patchwork-events-entity/src/main/java/com/patchworkmc/impl/event/entity/EntityEvents.java

```diff
@@ -87,6 +89,15 @@ public static float onLivingHurt(LivingEntity entity, DamageSource src, float da
 		return MinecraftForge.EVENT_BUS.post(event) ? 0 : event.getAmount();
 	}
 
+	public static float[] onLivingFall(LivingEntity entity, float distance, float damageMultiplier) {
+		LivingFallEvent event = new LivingFallEvent(entity, distance, damageMultiplier);
+		return MinecraftForge.EVENT_BUS.post(event) ? null : new float[]{ event.getDistance(), event.getDamageMultiplier() };
+	}
+
+	public static void onPlayerFall(PlayerEntity player, float distance, float damageMultiplier) {
```

Comment:
Make sure to clarify that onLivingFall is still fired for players that cannot fly.
```suggestion
	public static void onFlyablePlayerFall(PlayerEntity player, float distance, float damageMultiplier) {
```